### PR TITLE
Issue 19401 make the primary key of the identifier table as long as the dbs can handle as a primary key

### PR DESCRIPTION
**Pull request by nollymar**
Primary key length was increased to 750

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Task201016UpdatePrimaryKeyLengthIdentifierTable extends AbstractJDBCStartupTask {
+
+    @Override
+    public boolean forceRun() {
+        final Connection connection = DbConnectionFactory.getConnection();
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java:16]: Codacy found an issue: [Ensure that resources like this Connection object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        try {
+            final ResultSet resultSet = DotDatabaseMetaData.getColumnsMetaData(connection, "identifier");
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java:18]: Codacy found an issue: [Ensure that resources like this ResultSet object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTableTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task201016UpdatePrimaryKeyLengthIdentifierTableTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTableTest.java:14]: Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

**Review by jcastro-dotcms — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java:16]: Codacy found an issue: [Ensure that resources like this Connection object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java:18]: Codacy found an issue: [Ensure that resources like this ResultSet object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTableTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTableTest.java:14]: Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)

**Pull request merged**